clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -1648,6 +1648,26 @@ private static void checkMigrationStatus(CommandLine cmd) {
     ControllerClient destControllerClient = new ControllerClient(destClusterName, veniceUrl, sslFactory);
 
     ChildAwareResponse response = srcControllerClient.listChildControllers(srcClusterName);
+
+    // Check the migration status for system stores in the source and destination clusters
+    MultiStoreResponse srcClusterStores = srcControllerClient.queryStoreList();
```

Comment:
And then one last tweak.  Let's move these lines into the below if statements that check parent controller and child controllers (I guess by copying it twice or wrapping into a function, either works).

The logic right now for this tools is to

if (controllerUrl is NOT a parent controller) {
  only list migration status for this controllerUrl (one datacenter)
} else {
   list migration status for parent controller
   for() { list status for each child controller }
}

In the above flow we basically would like to execute your code for each datacenter (parent and each child) that we check.